.github/workflows/rust.yml

```diff
@@ -160,4 +160,69 @@ jobs:
       - name: cargo test fea-rs
         run: cargo test -p fea-rs
 
+  # Confirm we can still compile GS
+  build-gs:
+    runs-on: ubuntu-latest
+    name: Build GS
+    steps:
+      - name: Check out fontc source repository
+        uses: actions/checkout@v4
+      - name: Install the latest stable Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+      - name: Build and install fontc
+        run: cd fontc && cargo install --path . --debug
+      - name: Check out font project source repository
+        uses: actions/checkout@v4
+        with:
+          repository: googlefonts/googlesans
+          token: ${{ secrets.GS_READ_FONTC }}
+      - name: Download and unzip OTS v9.1.0
+        run: |
+            curl -OL "https://github.com/khaledhosny/ots/releases/download/v9.1.0/ots-9.1.0-Linux.zip"
+            unzip "ots-9.1.0-Linux.zip" "ots-9.1.0-Linux/ots-sanitize"
+      - name: Compile Roman variable font
+        run: fontc source/GoogleSans/GoogleSans.designspace
+      - name: OTS tests, Roman
+        run: ots-9.1.0-Linux/ots-sanitize build/font.ttf
+      - name: Compile Italic variable font
+        run: fontc source/GoogleSans/GoogleSans-Italic.designspace
+      - name: OTS tests, Italic
+        run: ots-9.1.0-Linux/ots-sanitize build/font.ttf
+
+  # Check if GS build is repeatable, https://github.com/googlefonts/fontc/issues/647
+  build-gs-repeatably:
```

Comment:
do we actually need a separate task for this? can't you add the extra builds at the end of `build-gs`?

Reply:
rebuilding GS should take a few seconds, bootstrapping again a whole container + rust toolchain + rebuilding fontc + checking out GS repo etc. will take much more

Reply:
We don't _need_ to separate, I just like getting separate checkmarks for builds GS at all vs builds it repeatably.

Reply:
The timing is interesting, looking at the new job and the old one:

* Build for debug takes :50
* Build for release takes 1:50
* Building GS with a debug compiler takes :50
* Building GS with a release compiler takes :06
* Checkout of source takes :14

All in all they both land around 2:40. I think will combine and use the release compiler, it should be basically always passing as soon as those bugs are fixed.

Reply:
So updated.